src/[sandbox]/examples/profile.ts

```diff
@@ -4,5 +4,5 @@ import { html } from "lit-element";
 
 export const profileTemplate = html`
   <h3>Default</h3>
-  <profile-view .profile=${profileMock} .presetTags=${presetTagsMock}> </profile-view>
+  <cjs-profile .profile=${profileMock} .presetTags=${presetTagsMock}> </cjs-profile>
```

Comment:
do you want to say cjaas? we would have to keep the "aa" - makes no sense to omit them

Reply:
That sounds good, I will do the same in my components